fsharp-backend/src/LibBackend/Pusher.fs

```diff
@@ -40,7 +44,7 @@ let push
   : unit =
   FireAndForget.fireAndForgetTask executionID $"pusher: {eventName}" (fun () ->
     task {
-      // TODO: handle messages over 10k
+      // TODO: handle messages over 10k (10k what? 10k bytes in size? does it fail after 10k (count) messages?)
```

Comment:
Pending question

Reply:
10k bytes in size. I think it fails if it's too big.